src/osgViewer/ViewerBase.cpp

```diff
@@ -205,11 +205,13 @@ void ViewerBase::setThreadingModel(ThreadingModel threadingModel)
 {
     if (_threadingModel == threadingModel) return;
 
+    bool needSetUpThreading = _threadsRunning;
```

Comment:
where did the changes in setThreadingModel come from?

Reply:
I don't see a change in either 3.6 or makOSG 3.4.   Bad auto-merge?   Reverted to the OSG 3.6 version.